File: Programmers/javascript/5.js
function gcd(a, b) {
    while (a > 0) {
        let temp = b;
        b = a;
        a = temp % a;
    }

    return b;
}

function solution(w, h) {
    return w * h - (w + h - gcd(w, h));
}

console.log(solution(8, 12));

File: Programmers/python/5.py
def solution(n):
    answer = ''
    num = ['4', '1', '2']

    while n > 0:
        answer = num[n % 3] + answer
        n = (n - 1) // 3

    return answer


print(solution(1))
print(solution(2))
print(solution(3))
print(solution(4))
